internal/store/model/alerts.go

```diff
@@ -7,20 +7,19 @@ import (
 )
 
 type Alert struct {
-	ID           uint64 `gorm:"primarykey"`
-	Provider     *Provider
-	ProviderID   uint64
-	ResourceName string
-	MetricName   string
-	MetricValue  string
-	Severity     string
-	Rule         string
-	TriggeredAt  time.Time
-	CreatedAt    time.Time
-	UpdatedAt    time.Time
+	ID           uint64    `db:"id"`
```

Comment:
@mabdh Will these models continue to exist in siren or go away in future PRs?

Reply:
That continue to exist as a placeholder from the DB result.